src/mpid/ch4/netmod/ofi/ofi_events.c

```diff
@@ -309,6 +309,13 @@ int MPIDI_OFI_send_event(struct fi_cq_tagged_entry *wc, MPIR_Request * sreq, int
         } else if (MPIDI_OFI_ENABLE_PT2PT_NOPACK && (event_id == MPIDI_OFI_EVENT_SEND_NOPACK))
             MPL_free(MPIDI_OFI_REQUEST(sreq, noncontig.nopack));
 
+        /* host buf != NULL means this request takes fall-back path
+         * for GPU data. */
```

Comment:
The term "fallback" is overloaded.  We use that when we fallback to AM-based implementations too.  Maybe call it "fallback to host-buffer staging"?